libs/ui/src/components/shared/card.tsx

```diff
@@ -0,0 +1,77 @@
+import { Avatar, Center, Heading, Image, Stack, Text } from '@chakra-ui/react';
+import { FC } from 'react';
+import { Navigate } from '../Navigate';
+
+interface CardProps {
+  title: string;
+  description: string;
+  image: string;
+  link: string;
+  rounded: boolean;
```

Comment:
```suggestion
  rounded?: boolean;
```

It could be optional